tests/integration/targets/omevv_firmware_compliance_info/tests/_hosts_helper.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Get group details
+  ansible.builtin.uri:
+    url: https://{{ hostname | ansible.utils.ipwrap }}:{{ port
+      }}/omevv/GatewayService/v1/Consoles/{{ uuid }}/Groups
+    user: "{{ vcenter_username }}"
```

Comment:
Use YAML anchors like &uri_params and reuse below instead of rewriting them again